### PR TITLE
Release FX Devices v1.0beta5

### PR DESCRIPTION
**Pull request by BryanChi**
-Add FX Layouts for some Valhalla plugins

### CODE REVIEW

**Comment by BryanChi** [comment]
So sorry to keep posting small updates everyday Cfillion, I'll try to keep the frequency down once the current version is stable..  
Or I'm thinking if I should have my own repository so I don't have to bother you every time I update..?

File: FX/BryanChi_FX Devices.lua [hunk 1]
```diff
@@ -36,6 +36,11 @@
 --   [effect] BryanChi_FX Devices/svf_filter.jsfx-inc
 --   BryanChi_FX Devices/IconFont1.ttf
 --   [effect] BryanChi_FX Devices/FXD (Mix)RackMixer
+--   [data] BryanChi_FX Devices/ValhallaVintageVerb (Valhalla DSP, LLC).ini > BryanChi_FX Devices/FX Layouts/ValhallaVintageVerb (Valhalla DSP, LLC).ini
+--   [data] BryanChi_FX Devices/ValhallaSupermassive (Valhalla DSP, LLC).ini > BryanChi_FX Devices/FX Layouts/ValhallaSupermassive (Valhalla DSP, LLC).ini
+--   [data] BryanChi_FX Devices/ValhallaDelay _Valhalla DSP_ LLC_.ini > BryanChi_FX Devices/FX Layouts/ValhallaDelay _Valhalla DSP_ LLC_.ini
+--   [data] BryanChi_FX Devices/ValhallaShimmer (Valhalla DSP, LLC).ini > BryanChi_FX Devices/FX Layouts/ValhallaShimmer (Valhalla DSP, LLC).ini
+--   [data] BryanChi_FX Devices/ValhallaFreqEcho (Valhalla DSP, LLC).ini > BryanChi_FX Devices/FX Layouts/ValhallaFreqEcho (Valhalla DSP, LLC).ini
```
Comment (cfillion) [inline FX/BryanChi_FX Devices.lua:43]: Using the "Data" type means these layout files would be installed under the &lt;resource path&gt;/Data directory. Seems that the script code expects to find them under Scripts:

```lua
local dir_path = ConcatPath(reaper.GetResourcePath(), 'Scripts', 'ReaTeam Scripts', 'FX', 'BryanChi_FX Devices' , 'FX Layouts')
```

Comment (BryanChi) [reply FX/BryanChi_FX Devices.lua:43]: I see. But how do I get it to be inside that directory?  For type I should choose ReaScript? But it doesn't seem to allow 
 /FX Layouts/filename.ini  as path name if I do so.

Comment (cfillion) [reply FX/BryanChi_FX Devices.lua:43]: > /FX Layouts/filename.ini

Remove the leading `/`: just "FX Layouts/filename.ini` in the "Storage file name" field.

Comment (BryanChi) [reply FX/BryanChi_FX Devices.lua:43]: Got it, thank you!

**Pull request closed without merging**